games/r.yaml

```diff
@@ -118,6 +118,24 @@
   type: remake
   updated: 2015-07-29
 
+- name: Rednukem
+  type: remake
+  originals:
+  - Redneck Rampage
```

Comment:
Also add `Duke Nukem 3D` since you can play this too with the very same executable.